### PR TITLE
Donate modal done

### PR DESCRIPTION
**Pull request by a-wong-8**

### CODE REVIEW

**Comment by a-wong-8** [comment]
Screenshot 2024-05-29 at 6 06 29 PM

**Review by a-wong-8 — COMMENTED** [review]
Screenshot 2024-05-29 at 6 06 29 PM

File: src/components/organisms/Navbar/Navbar.js [hunk 1]
```diff
@@ -71,45 +71,49 @@ export default function Navbar() {
...
-                                className="flex h-[35px] w-[35px] justify-center rounded-full bg-white px-1 py-2 font-bold text-black shadow-xl outline-none transition-all duration-150 ease-linear hover:bg-gray-300"
-                                type="button"
-                                onClick={() => setShowModal(false)}
-                            >
-                                X
-                            </button>
+                            {/* Close button */}
+                            <div className="absolute flex w-full flex-col items-end pt-3 pr-5">
+                                <button
+                                    className="flex justify-center rounded-full font-medium text-white text-xl shadow-xl outline-none transition-all duration-150 ease-linear"
```
Comment (gevuong) [inline src/components/organisms/Navbar/Navbar.js:92]: what does `transition-all duration-150 ease-linear` do to the button? I don't notice the effect

Comment (gevuong) [reply src/components/organisms/Navbar/Navbar.js:92]: oh it looks the only change that was made was the formatting.

**Review by gevuong — APPROVED** [review]

**Review by a-wong-8 — COMMENTED** [review]
thanks for the approval!

**Pull request merged**